packages/trip-analysis/README.md

```diff
@@ -1,31 +1,122 @@
+
+Our recommandation is to use [react-native-permissions](https://github.com/zoontek/react-native-permissions). You can find an implementation example in the [demo application inside this repository](../demo/App.tsx).
+
+#### Battery optimization
+
+In order to make Trip Analysis SDK to work properly, you need to disable battery optimization for your app: [native documentation](https://docs.drivequant.com/trip-analysis/android/get-started#battery-optimization)
+
+Our recommandation is to use [react-native-disable-battery-optimizations-android](https://github.com/rasheedk/react-native-disable-battery-optimizations-android).You can find an implementation example in the [demo application inside this repository](../demo/App.tsx).
+
+**IMPORTANT**
+
+This library is not actively maintained, but it does the job. To make it works you need to:
+
+1. Patch the `buile.gradle` file using `patch-package`. You can find the path we applied in our demo app [here](../../patches/react-native-disable-battery-optimizations-android%2B1.0.7.patch).
+2. Define the typescript interface, if needed. You can find the type definition in our demo app [here](../demo/src/types/react-native-disable-battery-optimizations-android.d.ts).
+
+This is the better solution we found. If you find a better solution, please feel free to open an issue/PR in this repository.
+
+---
+
+### iOS setup
+
+Call `initialize` method in your `AppDelegate.mm`.
+
+```objc
+// AppDelegate.mm
+#import <RNDriveKitTripAnalysis/react-native-drivekit-trip-analysis-umbrella.h>
 
 // ...
 
-const result = await multiply(3, 7);
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [[RNDriveKitCoreWrapper.shared initialize];
+  [RNDriveKitTripAnalysisWrapper.shared initializeWithLaunchOptions:launchOptions] // ADD THIS LINE
+}
 ```
 
-## Contributing
+**Note:** If you are using Swift, `initialize` method is also available.
+
+#### Configure capabilities
+
+You need to turn on Background Modes & enable Location updates. For more details please follow the [iOS documentation](https://docs.drivequant.com/trip-analysis/ios/get-started#configure-capabilities)
 
-See the [contributing guide](CONTRIBUTING.md) to learn how to contribute to the repository and the development workflow.
+#### Authorization
 
-## License
+In order to collect data, you need to configure multiple permissions :
 
-MIT
+- Location permission : [native documentation](https://docs.drivequant.com/trip-analysis/ios/get-started#ask-for-location-permission)
+- Motion permission : [native documentation](https://docs.drivequant.com/trip-analysis/ios/get-started#motion-and-fitness-permission)
+
+**Important:** Please note that `Motion` permission is only needed if you use [Motion & Fitness](https://docs.drivequant.com/trip-analysis/ios/get-started#motion-and-fitness-permission)
```

Comment:
Pour améliorer la reconnaissance de type de véhicule utilisé lors d'un trajet, notre SDK a besoin de cette permission. Des clients ne voulaient vraiment pas cette demande d'autorisation d'accès alors on explique dans notre [doc](https://docs.drivequant.com/trip-analysis/ios/get-started#motion-and-fitness-permission) qu'on peut désactiver, mais il est préférable de faire cette demande, et donc de supprimer cette note.